README.md

```diff
@@ -325,6 +335,11 @@ From gallery/, run `yarn start`.
 
 To view the gallery, point your browser to `http://localhost:4043/`.
 
+## Server-side Rendering Tests
+The `ssr-tests` top-level directory contains a simple next.js project which tests that RSC components can be used
+within next.js without causing server-side errors. Whenever a new component is created, these tests should be updated
+to include that new component. See `ssr-tests/README.md` for details.
+
 ## Continuous Integration and Releases
 This library has CI builds set up for both main and its dev branches. Builds of the main branch automatically get
 released as new versions. As part of this, dev branch builds have enforcement that they must increment the version
```

Comment:
Should we mention that we use semantic versioning? It's implied but not called out directly.

Reply:
> Should we mention that we use semantic versioning? It's implied but not called out directly.

It says so at the end of this paragraph.  However, also see ab79aff3c